crates/humanode-runtime/src/frontier_precompiles.rs

```diff
@@ -88,12 +90,11 @@ impl<R> Default for FrontierPrecompiles<R> {
     }
 }
 
-impl<R> FrontierPrecompiles<R>
-where
-    R: pallet_evm::Config,
-{
-    pub fn used_addresses() -> sp_std::vec::Vec<H160> {
-        sp_std::vec![
+impl<R: pallet_evm::Config> TypedGet for FrontierPrecompiles<R> {
+    type Type = Vec<H160>;
+
+    fn get() -> Self::Type {
+        [
```

Comment:
What about 
- using a wrapper over `FrontierPrecompiles` to pass it into `MigrationBrokenNoncesRecovery` instead of doing changes for `FrontierPrecompiles` itself to be satisfied getting precompiled addresses list ?
or
- just use hardcoded list of ignored contracts inside `evm-nonces-recover` crate without affecting  `FrontierPrecompiles` ?

Reply:
Thought about using `FrontierPrecompiles::used_addresses` as const generic param, but const generics can only be primitive types. Added `FrontierPrecompilesAddresses` wrapper.